src/IRsend.h

```diff
@@ -504,7 +504,7 @@ class IRsend {
                       const uint16_t nbytes,  // No default as there as so many
                                               // different sizes
                       const uint16_t repeat = kHitachiAcDefaultRepeat);
-#endif  // SEND_HITACHI_AC344
+#endif  // SEND_HITACHI_AC3
```

Comment:
Sigh. I suck sometimes. Nice catch!

Reply:
And me to, i missed it in the review and approved it 😢

Reply:
Don't feel bad if you miss something in review. It's my "job" as the code/PR author to get it right. If the reviewer catches it, it's a bonus!

See my (just added TL;DR:) to the code review section. https://github.com/crankyoldgit/IRremoteESP8266/wiki/Library-Maintainers-Guide#code-review Ref #115